lucene/test-framework/src/java/org/apache/lucene/codecs/uniformsplit/UniformSplitTestingCodec.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.lucene.codecs.uniformsplit;
+
+import org.apache.lucene.codecs.PostingsFormat;
+import org.apache.lucene.codecs.lucene50.Lucene50StoredFieldsFormat;
+import org.apache.lucene.codecs.lucene80.Lucene80Codec;
+
+/**
+ * Codec that chooses {@link UniformSplitPostingsFormat}. The problem with doing
+ * -Dtests.postingsformat=UniformSplitTesting instead is that it's awkward to disable this on some Lucene tests.
```

Comment:
I think we don't need this testing codec?  I read your javadoc comments here but there is no case yet where we want to disable uniformsplit (or any other particular PostingsFormat for that matter).

Reply:
Hum, you're right. I'll remove this Codec, yes.